src/test/java/nextstep/subway/acceptance/support/MemberSteps.java

```diff
@@ -91,6 +92,26 @@ public class MemberSteps {
                 .extract();
     }
 
+    public static ExtractableResponse<Response> 폼_로그인_후_내_회원_정보_조회_요청(String email, String password) {
+        return RestAssured
+            .given().log().all()
```

Comment:
AcceptanceTestSteps 에서 추가한 메서드를 재활용할 수 있을 것 같아요 😃 
```suggestion
        return given()
```